lunabot_embedded/firmware/update_board_manager.py

```diff
@@ -0,0 +1,33 @@
+import os
+import subprocess
+import tarfile
+import shutil
+import yaml
+
+# Step 4: Make the .arduino directory if it doesn't exist
+yaml_file_path = os.path.join(os.path.expanduser('~'), '.arduino15/arduino-cli.yaml')
```

Comment:
if __name__ == "__main__" this code? Didn't look at all files but same feedback if this exists in other places

Reply:
done